.github/workflows/node-ci-main.yml

```diff
@@ -51,17 +51,14 @@ jobs:
               run: yarn coverage
 
             - name: Upload Coverage
-              # We don't bother uploading a record of coverage for dependabot changes
-              if: ${{ github.actor != 'dependabot[bot]' }}
-              uses: coverallsapp/github-action@v1
+              uses: codecov/codecov-action@v3
               with:
-                  github-token: ${{ secrets.GITHUB_TOKEN }}
-                  flag-name: jest
-                  parallel: true
+                  token: ${{ secrets.CODECOV_TOKEN }}
+                  files: ./coverage/coverage-final.json
```

Comment:
@jeremywiebe should I set `continue-on-error: true` for this step?  I know some people had concerns about coverage drops show up as red `x`s on PRs.  Personally, I think we should be using required checks for those checks that we think are important enough to block merging.

Reply:
🤔  I don't think we should set `continue-on-error` to begin with. I'd rather tune the `target` value in our `codecov.yml` file if it's too sensitive.  

@handeyeco We can discuss and [disable this check](https://docs.codecov.com/docs/commit-status#informational) but still get the information even after Kevin lands this PR.

Reply:
Things should fall in the green zone with the current settings once the cypress coverage results are merged in.